src/apps/backend/modules/communication/communication-util.ts

```diff
@@ -0,0 +1,36 @@
+import { PhoneNumberUtil } from 'google-libphonenumber';
+
+import {
+  PhoneUtilInstance,
+  PhoneUtilInterface,
+  ValidationError,
+  ValidationFailure,
+} from './types';
+
+export default class CommunicationUtil {
```

Comment:
No need for this class.

Simply add validation of phone number when creating account.